src/utils/eyeToggle.js

```diff
@@ -0,0 +1,29 @@
+import { IconButton, InputAdornment } from '@material-ui/core';
+import { Visibility, VisibilityOff } from '@material-ui/icons';
+import React from 'react';
+
+export function toggleInputType(e, showPass) {
```

Comment:
Why do you export this function, is it used in some other files?

Reply:
It is used in several files.